From fda901d2d2f0b0934c02111bf8c294f20305eb12 Mon Sep 17 00:00:00 2001
From: xzl <xiangzelong@deepin.org>
Date: Mon, 10 Jul 2023 13:29:44 +0800
Subject: [PATCH] faketime-pdflatex

---
 doc/CMakeLists.txt | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/doc/CMakeLists.txt b/doc/CMakeLists.txt
index db898d0c..0cd9eceb 100644
--- a/doc/CMakeLists.txt
+++ b/doc/CMakeLists.txt
@@ -206,11 +206,11 @@ add_custom_target(doxygen_pdf
         COMMAND ${CMAKE_COMMAND} -E copy ${PROJECT_BINARY_DIR}/doc/doxygen_manual.tex  .
         COMMAND ${CMAKE_COMMAND} -E copy ${PROJECT_BINARY_DIR}/doc/manual.sty  .
         COMMAND ${CMAKE_COMMAND} -E copy ${PROJECT_SOURCE_DIR}/doc/doxygen_logo.pdf  .
-        COMMAND ${PDFLATEX}  -shell-escape doxygen_manual.tex
+        COMMAND faketime @$ENV{SOURCE_DATE_EPOCH} ${PDFLATEX} -shell-escape doxygen_manual.tex
         COMMAND ${MAKEINDEX} doxygen_manual.idx
-        COMMAND ${PDFLATEX}  -shell-escape doxygen_manual.tex
+        COMMAND faketime @$ENV{SOURCE_DATE_EPOCH} ${PDFLATEX} -shell-escape doxygen_manual.tex
         COMMAND ${MAKEINDEX} doxygen_manual.idx
-        COMMAND ${PDFLATEX}  -shell-escape doxygen_manual.tex
+        COMMAND faketime @$ENV{SOURCE_DATE_EPOCH} ${PDFLATEX} -shell-escape doxygen_manual.tex
         DEPENDS run_doxygen
         WORKING_DIRECTORY ${PROJECT_BINARY_DIR}/latex
 )
-- 
2.41.0

