From 13f5b6fee413d31404cf403a9855b7a9d257bc4f Mon Sep 17 00:00:00 2001
From: Dimitri van Heesch <doxygen@gmail.com>
Date: Tue, 23 May 2023 20:45:39 +0200
Subject: [PATCH] issue #10056 [1.9.7] Markdown Header ID ignored in
 subdirectories

---
 src/markdown.cpp | 10 +++++++---
 1 file changed, 7 insertions(+), 3 deletions(-)

diff --git a/src/markdown.cpp b/src/markdown.cpp
index 590d02b6c42..c0becf8dd3f 100644
--- a/src/markdown.cpp
+++ b/src/markdown.cpp
@@ -3403,10 +3403,14 @@ void MarkdownOutlineParser::parseInput(const QCString &fileName,
       }
       else
       {
-        if (title.isEmpty()) {title = titleFn;prepend=0;}
-        if (!wasEmpty)
+        if (title.isEmpty())
         {
-          docs.prepend("@ianchor{" + title + "} " +  markdownFileNameToId(fileName) + "\\ilinebr ");
+          title = titleFn;
+          prepend = 0;
+        }
+        if (wasEmpty)
+        {
+          docs.prepend("@ianchor{" + title + "} " + id + "\\ilinebr ");
         }
         else if (!generatedId.isEmpty())
         {
